docs/pom.xml

```diff
@@ -2584,7 +2584,6 @@
                             <goal>run</goal>
                         </goals>
                         <configuration>
-                            <jbangVersion>0.56.0</jbangVersion>
```

Comment:
is there a reason why we wouldn't lock it to a specific version?

Reply:
No reason, except that it doesn't need to be changed when there are fixes in jbang